cmd/bolt-agent/main.go

```diff
@@ -219,6 +219,12 @@ func getApp() *cli.App {
 			Usage:  "do not report on-chain balance",
 			Hidden: true,
 		},
+		&cli.DurationFlag{
+			Name:   "syncedtochain-cooldown",
+			Usage:  "how long to wait before reporting node is not synced to chain",
```

Comment:
I would add info why this is needed. Explain that zmq might have network delay while block height is bigger...